model/music.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+require_relative './item'
+class Music < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify:)
+    super(publish_date: Date.today.to_s)
+    @on_spotify = on_spotify
+  end
+
+  def to_json(*_args)
+    JSON.dump(
+      on_spotify: @on_spotify
+    )
+  end
+
+  private
+
+  def can_be_archived?
+    return true if super && @on_spotify
+  end
+end
```

Comment:
- [ ] You are almost there! However, I was not able to add a music album, this is what it shows:

<img width="331" alt="Screen Shot 2022-07-30 at 1 40 05 PM" src="https://user-images.githubusercontent.com/83717285/181937399-1639f4f7-0e8b-43ea-9e6c-9224fb2dfe3f.png">

After saying 'yes' or 'no' to spotify it says 'created!' and this is what's inside the JSON file:

<img width="900" alt="Screen Shot 2022-07-30 at 1 40 50 PM" src="https://user-images.githubusercontent.com/83717285/181937457-31f04490-c84b-448a-8028-17eda93e06c5.png">

Therefore, this is the music albums list:

<img width="301" alt="Screen Shot 2022-07-30 at 1 49 52 PM" src="https://user-images.githubusercontent.com/83717285/181937525-f1b2ccf9-058a-4050-9245-d2ce89dd9388.png">

It would be great to fix that bug, you are almost done! 🤓

Reply:
Thanks for the review changes made